package.json

```diff
@@ -157,6 +157,7 @@
     "test:fmt": "gulp testfmt",
     "test:lang": "gulp testlang",
     "update": "gulp update",
-    "watch-streamer": "cd docs/static/streamer && tsc -t es6 --watch"
+    "watch-streamer": "cd docs/static/streamer && tsc -t es6 --watch",
+    "postinstall": "cd skillmap && npm install && cd .. && cd authcode && npm install && cd .."
```

Comment:
If i recall correctly the `npm ci` was there as it sped up builds quite a bit - is that still the case / something we should consider?

Reply:
Oh, makes sense. I'll put it back.

Reply:
Would be nice if `postinstall` had a ci-specific variant.

Reply:
@riknoll did `npm ci` for skillmap speed up the build? Wondering whether it's worth the tradeoff of having to manually install dependencies for skillmap, authcode, and soon multiplayer.

Reply:
One thing you can do is just have a postinstall.js that checks prod vs normal and execs the correct installs / cis, and have postinstall run `node postinstall.js` (would just have to copy over the existing code from cli, we set some env variables for ci builds)

Reply:
I'd like to merge this, then improve postinstall as you suggest in a separate PR.